server/src/main/java/org/eclipse/openvsx/UserAPI.java

```diff
@@ -224,7 +224,7 @@ public CsrfTokenJson getCsrfToken(HttpServletRequest request) {
                 String url = createApiUrl(serverUrl, "api", namespace.getName(), ext.getName());
                 json.extensions.put(ext.getName(), url);
             }
-            json.access = NamespaceJson.RESTRICTED_ACCESS;
+            json.verified = true;
```

Comment:
should we not still keep setting the `access` property for compatibility reasons?

Reply:
Yes, I kept it for the public API (`RegistryAPI`), which is documented at https://open-vsx.org/swagger-ui/. This part of the API is used by the web UI and is regarded as internal (it requires authentication).